src/main/kotlin/no/nav/klage/oppgave/service/BehandlingService.kt

```diff
@@ -449,6 +449,18 @@ class BehandlingService(
         }
     }
 
+    @Transactional(readOnly = true)
+    fun getBehandlingForSmartEditor(behandlingId: UUID, utfoerendeSaksbehandlerIdent: String): Behandling {
+        val behandling = behandlingRepository.findById(behandlingId).get()
+        if (behandling.currentDelbehandling().medunderskriver?.saksbehandlerident == utfoerendeSaksbehandlerIdent) {
+            checkMedunderskriverStatus(behandling)
```

Comment:
Jeg merker meg at denne ikke sjekker det samme som "checkSkrivetilgang for saksbehandler". Der sjekkes også om saken er avsluttet eller ikke. D.v.s. er to ulike sjekker i én. Burde gjelder medunderskriver også. Er innlogget bruker saksbehandler eller medunderskriver, og er saken avsluttet slik at de har skrivetilgang.